server_app/user/handler.js

```diff
@@ -117,5 +145,7 @@ module.exports = router
 	.post("/login", userLogin)
 	.post("/recovery", userRecovery)
 	.post("/reset", resetPassword)
+	.post("/updatePassword", updatePassword)
```

Comment:
I think we could keep the standard "/update/password" 

the same for the another endpoint